guestbook.module

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/**
+* @file
+* Contains hooks for guestbook module.
+*/
+
+module_load_include('inc', 'guestbook', 'include/guestbook.form');
+module_load_include('inc', 'guestbook', 'include/guestbook.core');
+module_load_include('inc', 'guestbook', 'include/guestbook.theme');
+
+/**
+ * URL for downloading xml weather file.
+ */
+define('WEATHER_URL', 'https://xml.meteoservice.ru/export/gismeteo/point/2897.xml');
+
+/**
+ * Implements hook_permission().
+ */
+function guestbook_permission() {
+  return array(
+    'access guestbook messages' => array(
+      'title' => t('Show messages for the Guestbook module'),
+    ),
+    'access guestbook content' => array(
+      'title' => t('Access to edit and delete for the Guestbook module'),
+    ),
+  );
+}
+
+/**
+* Implements hook_menu(). 
+*/
+function guestbook_menu() {
+  // Main page.
+  $items['guestbook-page'] = array(
+    'title' => 'Guestbook',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('guestbook_page_form'),
+    'access arguments' => array('access guestbook messages'),
+  );
+  // Edit message.
+  $items['guestbook-page/%guestbook_mid/edit'] = array(
+    'title' => 'Guestbook',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('guestbook_page_form', 1),
+    'access arguments' => array('access guestbook content'),
+  );
+  // Delete message.
+  $items['guestbook-page/%guestbook_mid/delete'] = array(
+    'title' => 'Delete message',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('guestbook_message_form_delete', 1),
+    'access arguments' => array('access guestbook content'),
+  );
+
+  return $items;
+}
+
+/**
+ * Loads a message info.
+ *
+ * @param $mid
+ *   Integer specifying the message id to load.
+ *
+ * @return array
+ *   A fully-loaded array of message info ('id', 'uid', 'sid', 'date', 'message').
+ *   id - message id; uid - user id; sid - session id; date - message creation
```

Comment:
@PavelSPN this annotation break into the new lines like
` - id: message ID;`
`- uid: user ID;`
and so on